opendg/_storage/base.py

```diff
@@ -72,3 +66,9 @@ def num_edges(self) -> int:
     @abstractmethod
     def num_timestamps(self) -> int:
         r"""The total number of unique timestamps encountered over the temporal graph."""
+
+    def _check_slice_time_args(self, start_time: int, end_time: int) -> None:
```

Comment:
Will continue to add in helper functions that will be common across backends into the base class (including device mixins, etc.)